.pipelines/templates/.builder-release-template.yaml

```diff
@@ -76,6 +76,7 @@ steps:
         -e AZURE_VM_SIZE=$(AZURE_VM_SIZE) \
         -e AZURE_RESOURCE_GROUP_NAME=${AZURE_RESOURCE_GROUP_NAME} \
         -e AZURE_LOCATION=${AZURE_LOCATION} \
```

Comment:
is this VAR still used ?  what would be a use case for having a differnet AZURE_LOCATION and PACKER_LOCATION

Reply:
I just hacked this up to see if this stuff even works, which it looks like does so far...

will revisit after further testing :)